work/v1/types.go

```diff
@@ -46,6 +51,56 @@ type ManifestsTemplate struct {
 	Manifests []Manifest `json:"manifests,omitempty"`
 }
 
+type DeleteOption struct {
+	// propagationPolicy can be Orphan or SelectivelyOrphan
```

Comment:
I think Foreground needs to be an option too, right?  That way I can have this stanza and explicitly indicate my desire for the current default.